inc/RestApi/SettingsController.php

```diff
@@ -184,6 +184,8 @@ public function get_current_settings() {
 			'hasSetHomepage'          => (bool) get_option( 'bh_has_set_homepage', false ),
 			'showOnFront'             => (string) get_option( 'show_on_front' ),
 			'pageOnFront'             => (int) get_option( 'page_on_front' ),
+			// Can be setup similar to BH under 'wordpress'
+			'isWooActive'             => class_exists( 'woocommerce' ) ? 1 : 0,
```

Comment:
What does this code comment mean?

Reply:
It doesn't really feel like it belongs in the settings - since it's not something a user can change through our plugin like these other settings, these are more for the options they can manage via the controls in the plugin, we'll never need to write this value so it doesn't need to be exposed in the api like this. 

I think we should put it into the `inc/Data.php` values. Then it will be in the js on the plugin app page, like we do for `resturl` https://github.com/newfold-labs/wp-plugin-crazy-domains/blob/main/inc/Data.php#L27 and then pass into the marketplace component https://github.com/newfold-labs/wp-plugin-crazy-domains/blob/main/src/app/pages/marketplace/index.js#L47